tests/functional/test_direct_edit.py

```diff
@@ -147,6 +147,68 @@ def test_cleanup_bad_folder_name(direct_edit):
     assert folder.is_dir()
 
 
+def test_cleanup_orphan_files(manager_factory, obj_factory):
+    """
+    Files marked as locked must be ignored by _cleanup.
+    These files are removed after there are unlocked.
+    """
+    manager, engine = manager_factory()
+
+    with manager:
+        direct_edit = manager.direct_edit
+        direct_edit._folder.mkdir()
+        doc = obj_factory(title="this is a orphan file.txt", nature="File")
```

Comment:
Just for the completeness of the test, do not use ".txt" as they will be seen as a Note. Use ".odt" instead:
```suggestion
        doc = obj_factory(title="this is a orphan file.odt", nature="File")
```